src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long
 
-__version__ = "2.34.1.1"
+__version__ = "2.34.1"
```

Comment:
We don't need `.1` anymore since https://github.com/Azure/azure-cli/pull/19079 guarantees the uniqueness of versions of packages built from non-`release` branches.